pages/plugins.jsx

```diff
@@ -1,15 +1,53 @@
 import React from 'react'
 
+import Plugin from '../components/plugin'
+import pluginsClassNames from './plugins.css'
+
+const integrations = [
+  {
+    name: 'Analytics',
+    abstract:
+      'Enhance your statistics by enabling collecting data from your website',
+    description:
+      'Increase your website discoverability by cross-linking similar papers with help of our powerful recommender system',
+  },
+  {
+    name: 'Discovery',
+    abstract:
+      'Enrich your full text amount by dynamic adding missing bits from CORE',
+    description:
+      'Increase your website discoverability by cross-linking similar papers with help of our powerful recommender system',
+  },
+  {
+    name: 'Reader',
+    abstract: 'Enable future reading experience for the files on your website',
+    description:
+      'Increase your website discoverability by cross-linking similar papers with help of our powerful recommender system',
+  },
+  {
+    name: 'Recommender',
+    abstract:
+      'Increase your website discoverability by cross-linking similar papers',
+    description:
+      'Increase your website discoverability by cross-linking similar papers with help of our powerful recommender system',
+  },
+]
+
 const Plugins = () => {
   return (
     <>
-      <h1>Dashboard plugins tab</h1>
-      <p>
-        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Mollitia
-        numquam aliquid, blanditiis praesentium doloribus molestiae sequi facere
-        quasi minima odio officia maxime accusantium iusto ducimus labore ut
-        nemo perferendis libero.
-      </p>
+      <h1>Plugins</h1>
+      <p>Integrate our powerful services into your website.</p>
+      <div className={pluginsClassNames.plugins}>
+        {integrations.map(i => (
+          <Plugin
+            key={i.name}
+            name={i.name}
+            abstract={i.abstract}
+            description={i.description}
+          />
+        ))}
```

Comment:
I am not a big fan of spread syntax because of potential bugs.

I still haven't decided whether we should have separate components for every plugin or rather to have one general component with the config. I rather incline to separate components. Not sure... What do you think?

Reply:
For the first time a generic component fits our needs. We cannot develop as much as in the design so we will have only installation instructions inside each plugin, and basically only 2 plugins.

On the other hand, you have a good point. Plugins have different configuration options and installation steps.